### ISSUE TITLE
As a SO/CE I would like to create number elements that when the user loads the page that it will count up to a specific number

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/express#895]
## Context

A design trend that has been requested is the ability to have a number count up when the user loads the page, or scrolls to that element on the page.

http://bfintal.github.io/Counter-Up/demo/demo.html

## Possible Solution

Creating a shortcode for these elements provides not only an easy way to format these elements, but the ability to use them anywhere in a site that has a WYSWYG text editor.

**Comment by cathysnider** [comment]
Acceptance Criteria (pre-merge)
- [ ] numbers encased in the countup shortcode do countup 
- [ ] in all WYSIWYG containers (i.e. articles, basic pages, blocks, content grids)
- [ ] other shortcodes continue working

https://www-dev.colorado.edu/p1e3e27f8eb6/

Test Results: Fail. The Column Shortcode no longer works:  https://www-dev.colorado.edu/p1e3e27f8eb6/your

There's also a weird case where, when the number is 123456789, the last number gets stuck on eight: https://www-dev.colorado.edu/p1e3e27f8eb6/drupal/sub-page-one

**Comment by cathysnider** [comment]
Notes on testing sites on www-test: 
- Column shortcode DOES work
- Counting-up time does NOT work
- Only one Expand link gets to count-up
